simulation-construction-set-examples/simple-pendulum-with-controller/src/main/java/us/ihmc/simplePendulum/SimplePendulumController.java

```diff
@@ -29,12 +33,15 @@ public class SimplePendulumController implements RobotController
    private double torque;
 
    // Constructor: Where we instantiate and initialize control variables
-   public SimplePendulumController(SimplePendulumRobot robot)
+   public SimplePendulumController(ControllerInput controllerInput, ControllerOutput controllerOutput)
    {
-      this.robot = robot;
       desiredPositionRadians = new YoDouble("DesiredPosRad", registry);
       desiredPositionRadians.set(-1.5); // set initial position of the pendulum
 
+      // Get the objects of the fulcrum joint to read input and write output in the control loop
+      this.fulcrumJoint = (OneDoFJointReadOnly) controllerInput.getInput().findJoint("FulcrumPin");
```

Comment:
Let's point to a new static String for the joint name that would be defined in the robot defintion

Reply:
Done.